app/Services/Notification/config.ts

```diff
@@ -0,0 +1,90 @@
+import { Integration, NotificationTemplate } from './types'
```

Comment:
I think this file should be in the `config` directory

Reply:
It's called config but I'll rename to `defaults`, makes more sense

Reply:
I still think this is intended to be in the `config` directory
Have a look at the defaults files from Adonis, there're all in this directory...

Reply:
Okay I'll do it!